src/Avalonia.Base/Media/FormattedText.cs

```diff
@@ -50,13 +50,15 @@ public class FormattedText
         /// <param name="typeface">Type face used to display text.</param>
         /// <param name="emSize">Font em size in visual units (1/96 of an inch).</param>
         /// <param name="foreground">Foreground brush used to render text.</param>
+        /// <param name="features">Optional list of turned on/off features.</param>
         public FormattedText(
             string textToFormat,
             CultureInfo culture,
             FlowDirection flowDirection,
             Typeface typeface,
             double emSize,
-            IBrush? foreground)
+            IBrush? foreground,
+            FontFeatureCollection? features = null)
         {
```

Comment:
I can't review this PR properly, will leave it to @Gillibald .
But adding new arguments to the methods is a binary breaking change, which we can't accept until 12.0, which is a long time from now. I suppose we can have a second constructor instead.

CI build failed because of that as well https://dev.azure.com/AvaloniaUI/AvaloniaUI/_build/results?buildId=43426&view=results

Reply:
@maxkatz6 I added constructors that allow to pass binary compatibility and updated initial comment. Because almost all parameters are optional, usage of TextLayout, GenericTextRunProperties and TextShaperOptions constructors outside Avalonia sources may give ambiguous constructor compile time error.